src/components/inline-link/types.ts

```diff
@@ -25,4 +21,9 @@ export interface InlineLinkProps {
    * The `weight` passed to the inner `Text` component.
    */
   textWeight?: TextProps['weight']
+
+  /**
+   * The text to render in the anchor
```

Comment:
🚧 Can this be updated for accuracy and to still mention <a> element?

```suggestion
   * The content to render within the `<a>` element.
```